### ISSUE TITLE
We need a test plugin for xplanemp

### ISSUE THREAD

**Issue by kuroneko** [kuroneko/libxplanemp#14]
We actually need to sit down and write a test-plugin that uses xplanemp for three goals:

* Demonstration of how to use the API.
* To allow CSL authors to test their CSLs offline
* To allow us to test the library in a controlled environment.

**Comment by lucyydotp** [comment]
I agree with this. I can't find any documentation on how to use this thing anywhere.

**Comment by kuroneko** [comment]
The only API documentation right now is in the headers in `include` themselves.  The basic operational model is callback driven.
